contracts/rewards/src/state.rs

```diff
@@ -168,18 +241,18 @@ pub trait Store {
 /// Current rewards parameters, along with when the params were updated
 pub const PARAMS: Item<StoredParams> = Item::new("params");
 
-/// Maps a (contract address, epoch number) pair to a tally for that epoch and contract
-const TALLIES: Map<(Addr, u64), EpochTally> = Map::new("tallies");
+/// Maps a (pool id, epoch number) pair to a tally for that epoch and rewards pool
+const TALLIES: Map<TallyKey, EpochTally> = Map::new("tallies");
 
-/// Maps an (event id, contract address) pair to an Event
-const EVENTS: Map<(String, Addr), Event> = Map::new("events");
+/// Maps an (event id, pool id) pair to an Event
+const EVENTS: Map<(String, PoolId), Event> = Map::new("events");
```

Comment:
might be worthwhile to use a type alias for String here, to make things clearer.

Reply:
Agree, can do in another pr. There some other places using event id.